### DISCUSSION TITLE
TP5 - uso de scanf en funciones

### DISCUSSION THREAD

**Discussion by rinrot** [INGCOM-UNRN-P1/cursada-2023#153] [Preguntas y respuestas] [answered]
Hola! estaba haciendo el ejercicio 1 del tp5 y queria usar una funcion para cargar los valores de las matrices para que haya menos codigo en el main. no tendria que haber ningun problema no?

**Comment by martinvilu** [comment, marked as answer]
Termina siendo lo mismo que `cargar_arreglo` del TP3.

**Comment by martinvilu** [comment]
Duplicado de: https://github.com/INGCOM-UNRN-P1/cursada-2023/discussions/156
